shared/src/userData.ts

```diff
@@ -27,9 +27,11 @@ export interface Business {
   readonly preferences: Preferences;
   readonly taxFilingData: TaxFilingData;
   readonly formationData: FormationData;
+  readonly version: number;
 }
 
 export const CURRENT_VERSION = 147;
+export const CURRENT_BUSINESS_VERSION = 1;
```

Comment:
we are updating this to 147? Can we reference the line above?

Reply:
This was the older implementation. We are not referencing CURRENT_BUSINESS_VERSION going forward